python/meep.i

```diff
@@ -1410,6 +1449,9 @@ void _get_gradient(PyObject *grad, PyObject *fields_a, PyObject *fields_f, PyObj
 }
 
 %exception {
+#ifdef MEEP_SWIG_PYTHON_DEBUG
```

Comment:
Is it worth having a `MEEP_SWIG_PYTHON_DEBUG` just for this `printf`, vs. just adding a `printf` when you need it?  What subsequent developer is likely to need a `printf` in `_get_gradient`?

Reply:
I am happy to remove it if you think it is just noise.
I found this useful to track down which SWIG wrapped methods were taking a lot of time etc.